README.md

```diff
@@ -337,329 +337,3 @@ Introduction to the Splunk product and some of its capabilities
 ## Contact
 
 You can reach the Dev Platform team at [devinfo@splunk.com](mailto:devinfo@splunk.com).

```

Comment:
I'll figure that part out.  But this particular removal was due to duplicate sections in `README.md` in the first place.